thirdparty/DiskANN/include/aux_utils.h

```diff
@@ -99,7 +99,7 @@ namespace diskann {
   template<typename T>
   DISKANN_DLLEXPORT int build_merged_vamana_index(
       std::string base_file, diskann::Metric _compareMetric, unsigned L,
-      unsigned R, double sampling_rate, double ram_budget,
+      unsigned R, bool fast, double sampling_rate, double ram_budget,
```

Comment:
Can we name it in a more clear way? Such as `accelerate_build`, etc. PS, this is just an example.